docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -92,6 +92,7 @@ supports the following features:
 * :doc:`/sql/delete`, see also :ref:`iceberg-delete`
 * :ref:`sql-schema-table-management`, see also :ref:`iceberg-tables`
 * :ref:`sql-materialized-views-management`
+* :ref:`sql-views-management`
```

Comment:
Should this be in Hive connector too?

Reply:
Yes ... we have to add the SQL support section to a lot of connectors still. Iceberg, sqlserver and postgresql were just the start..